2017-bae-europe/eu_future/eu_future.ipynb

```diff
@@ -137,10 +133,164 @@
   },
   {
    "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total Tails in MTHM:\n",
+      "1081826.9578\n"
+     ]
+    }
+   ],
+   "source": [
+    "# get tailings timeseries both cum and non-cum\n",
+    "cur = an.get_cursor('hope.sqlite')\n",
+    "\n",
+    "init_year, init_month, duration, timestep = an.get_timesteps(cur)\n",
+    "\n",
+    "agent_ids = an.get_agent_ids(cur, 'Sink')\n",
+    "tails_dict = an.facility_commodity_flux(cur, agent_ids, ['tails'], False)\n",
+    "tails_no_cum = an.facility_commodity_flux(cur, agent_ids, ['tails'], False, False)\n",
+    "an.double_axis_bar_line_plot(tails_no_cum, tails_dict, timestep, 'Years', 'Monthly Discharge Mass [MTHM]', 'Cumulative Mass [MTHM]',\n",
+    "                    'Tails Mass vs Time', 'tailings_cum_no_cum', init_year)\n",
+    "\n",
+    "print('Total Tails in MTHM:')\n",
+    "print(max(tails_dict['tails']))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "done\n"
+     ]
+    }
+   ],
+   "source": [
+    "# get fuel timeseries both cum and non-cum\n",
+    "\n",
+    "cur = an.get_cursor('hope.sqlite')\n",
+    "\n",
+    "init_year, init_month, duration, timestep = an.get_timesteps(cur)\n",
+    "\n",
+    "uox_dict = an.fuel_usage_timeseries(cur, ['uox'], True)\n",
+    "uox_dict_no_cum = an.fuel_usage_timeseries(cur, ['uox'], False)\n",
+    "an.double_axis_bar_line_plot(uox_dict_no_cum, uox_dict, timestep, 'Years',  'Monthly Discharge Mass [MTHM]','Cumulative Mass [MTHM]',\n",
+    "                    'UOX Mass vs Time', 'uox_cum_to_no_cum', init_year)\n",
+    "\n",
+    "mox_dict = an.fuel_usage_timeseries(cur, ['mox'], True)\n",
+    "mox_dict_no_cum = an.fuel_usage_timeseries(cur, ['mox'], False)\n",
+    "an.double_axis_bar_line_plot(mox_dict_no_cum, mox_dict, timestep, 'Years',  'Monthly Discharge Mass [MTHM]','Cumulative Mass [MTHM]',\n",
+    "                    'MOX Mass vs Time', 'mox_cum_to_no_cum', init_year)\n",
```

Comment:
If a plot is created, especially for analysis reasons, can you show it in the note as well?